server/svix-server/src/queue/redis.rs

```diff
@@ -1,31 +1,43 @@
-/// Redis queue implementation
-///
-/// This implementation uses the following data structures:
-/// - A "tasks to be processed" queue - which is what the consumer listens to for tasks.
-///     AKA: Main
-/// - A "tasks currently processing" queue - which are for tasks that are currently being handled.
-///     AKA: Processing
-/// - A ZSET for delayed tasks with the sort order being the time-to-be-delivered
-///     AKA: Delayed
-///
-/// - Tasks in the queues are prefixed with a ksuid so that we can know the timestamp of when they
-/// should be executed.
-///
-/// The implementation spawns an additional worker that monitors both the zset delayed tasks and
-/// the tasks currently processing. It monitors the zset task set for tasks that should be
-/// processed now, and the currently processing queue for tasks that have timed out and should be
-/// put back on the main queue.
+//! Redis stream-based queue implementation
```

Comment:
Why not `///`? I don't even remember what `//!` does tbh.

Reply:
It's just the convention for documenting modules in whole. See https://doc.rust-lang.org/reference/comments.html